### DISCUSSION TITLE
Ideas for Repository structure organization & readme file formats

### DISCUSSION THREAD

**Discussion by saurabh47** [saurabh47/Popular_Coding_Questions_Solution#28] [Ideas]
Currently our repository is not properly organized. we should have some readme file with link to specific program file
For reference checkout this repository https://github.com/AnasImloul/Leetcode-Solutions
All the programs file are organized by alphabetical order at root level & After clicking on that there is readme file containing table with proper link to the program.

**Comment by saurabh47** [comment]
@maheshmnj Any thoughts on this

**Comment by maheshj01** [comment]
Yes we can track this under https://github.com/saurabh47/Popular_Coding_Questions_Solution/issues/24
